wiki/en/en-Installation-for-Linux.md

```diff
@@ -14,18 +14,18 @@ Although we don't provide official packages for every Linux distribution, you mi
 
 [![Packaging status](https://repology.org/badge/vertical-allrepos/jamulus.svg)](https://repology.org/project/jamulus/versions)
 
-There is also list of unofficial [packages for different distributions here](https://github.com/corrados/jamulus/issues/223).
+There is also list of unofficial [packages for different distributions here](https://github.com/jamuluswebsite/jamulus/issues/223).
 
 ### Debian and Ubuntu (amd64 .deb only)
 
 If you're on amd64 **Debian**/**Ubuntu**, you may try the compiled .deb packages from GitHub Actions.
 
-1. Download the jamulus_latest_ubuntu_amd64.deb file from the [Jamulus GitHub release](https://github.com/corrados/jamulus/releases/tag/latest) page
+1. Download the jamulus_latest_ubuntu_amd64.deb file from the [Jamulus GitHub release](https://github.com/jamuluswebsite/jamulus/releases/tag/latest) page
```

Comment:
We'll probably need to update this link once the release is made. Also `apt --fix-broken install` is needed on a clean machine. (Not sure if there's a cleaner way to install the dependencies...)

Reply:
Agree. #287 might be something we should include here? Probably the installation on Linux needs to be rewritten.

Reply:
The Linux installation stuff needs to be re-done now that we're in the repos properly, with compilation instructions relegated to a file in the repo I think (and just linked to in passing from the website). 

But in the interests of time, can we keep as is? Release is coming soon...